src/app/admin/components/equipment/equipment.component.html

```diff
@@ -0,0 +1,214 @@
+<lc-modal-layout [headerTitle]="modalHeader">
+  <ng-container body>
+    <form *ngIf="form" [formGroup]="form">
+      <div class="row">
+        <div class="column">
+          <mat-form-field appearance="fill" floatLabel="auto">
+            <mat-label>Название</mat-label>
```

Comment:
better to have it as const or enum. Same for others

Reply:
fixed

Reply:
fixed